synergy-design-system/synergy-design-system
# fix: 🤔 Support Angular@18

# Pull Request

## 📖 Description

This PR adds support for `angular@18` in `@synergy-design-system/angular`. It also comes with some additional changes in the demo:

- The demo project is upgraded to `angular@18`
- Routing is now working using SPA routes via angular router as discussed in https://github.com/synergy-design-system/synergy-design-system/issues/442.

### 🎫 Issues

Closes #401

## 👩‍💻 Reviewer Notes

Please have a look at the changed files

## 📑 Test Plan

- Check out this branch
- Install the latest dependencies
- Do a full build `pnpm build` in the root
- Spin up the angular demo
- Play around with the routing and form functionality
- Run the e2e tests in `packages/_private/e2e-demo-test`

## ✅ DoD
